pkg/cloudprovider/vsphere/nodemanager_test.go

```diff
@@ -126,6 +153,7 @@ func TestDiscoverNodeByName(t *testing.T) {
 
 	vm := simulator.Map.Any("VirtualMachine").(*simulator.VirtualMachine)
 	name := vm.Config.GuestFullName
+	fmt.Println("Name:", name)
```

Comment:
`t.Logf` instead, gives filename:linenumber and only shows up with `go test -v`

Reply:
See first comment above.